slides/python.qmd

```diff
@@ -245,21 +244,21 @@ Get in touch:
 {{< fa brands mastodon >}} \ [\@jatkinson1000\@fosstodon.org](https://fosstodon.org/@jatkinson1000)
 :::
 ::: {.column width="35%"}
-{{< fa pencil >}} \ Amy Pike
-
-{{< fa solid envelope >}} \ [ap766[AT]cam.ac.uk](mailto:ap766@cam.ac.uk)
-
-{{< fa brands github >}} \ [AmyOctoCat](https://github.com/AmyOctoCat)
-
-\
-
 {{< fa pencil >}} \ Marion Weinzierl
 
 {{< fa solid envelope >}} \ [mw925[AT]cam.ac.uk](mailto:mw925@cam.ac.uk)
 
 {{< fa brands github >}} \ [MarionBWeinzierl](https://github.com/MarionBWeinzierl)
 
 {{< fa brands mastodon >}} \ [\@MarionBWeinzierl\@mast.hpc.social](https://mast.hpc.social/@MarionBWeinzierl)
+
```

Comment:
This change is just to that the full contact info for everyone is visible and not hidden by the logo